src/game/logic/object/object.h

```diff
@@ -274,7 +275,7 @@ class Object : public Thing, public SnapShot
     void Clear_Status(BitFlags<OBJECT_STATUS_COUNT> bits) { return Set_Status(bits, false); }
     bool Clear_Script_Status(ObjectScriptStatusBit bit) { return Set_Script_Status(bit, false); }
 
-    Module *Find_Module(NameKeyType type) const;
+    BehaviorModule *Find_Module(NameKeyType type) const;
```

Comment:
Are you sure this shouldn't just become `Find_Behaviour_Module` like the `Find_Damage_Module` and `Find_Update_Module`